docs/community.aws.aws_acm_info_module.rst

```diff
@@ -27,8 +27,8 @@ Requirements
 The below requirements are needed on the host that executes this module.
 
 - python >= 3.6
-- boto3 >= 1.16.0
-- botocore >= 1.19.0
+- boto3 >= 1.17.0
+- botocore >= 1.20.0
```

Comment:
This must not be changed.  
I guess you must remove your local amazon.aws > 4/main and rebuild docs with amazon.aws 3.3.1/stable-3